### PR TITLE
Test calculator: Testing Math-Magician website

### PR DESCRIPTION
**Pull request by Ceejayski**
In this project:

- Write unit tests for calculate.js and operate.js

- Use snapshots and react-testing-libraries to test App function.

- Use snapshots and create from react-test-renderer to test Button, ButtonPanel functions.

### CODE REVIEW

File: src/_test_/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  expect(operate('4', '5', '-')).toEqual('-1');
+});
+
+test('can multiply numbers', () => {
+  expect(operate('4', '5', 'x')).toEqual('20');
+});
+
+test('can divide numbers', () => {
+  expect(operate('4', '5', '/')).toEqual('0.8');
+});
```
Comment (addod19) [inline src/_test_/operate.test.js:17]: ### Optional
- [ ] It would be better if we add a test for the division by zero logic. For example; ` 10 / 0` should return an error message

File: src/_test_/App.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { create } from 'react-test-renderer';
```
Comment (addod19) [inline src/_test_/App.test.js:1]: - [ ] It would be better if we update the README to include instructions about how to run the test, title and description

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @Ceejayski, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/_test_/operate.test.js (hunk 1 above)
Comment (addod19) [reply src/_test_/operate.test.js:17]: - [ ] It would be better if we fix the 2 obsolete tests. We need to pass all the test

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @Ceejayski, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
